dev-libs/ferrisloki/ferrisloki-3.0.13-r3.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic
+
+DESCRIPTION="Loki C++ library from Modern C++ Design"
+HOMEPAGE="http://www.libferris.com/"
+SRC_URI="mirror://sourceforge/witme/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+RDEPEND=">=dev-libs/libsigc++-2.6:2"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.0.13-r3-configure-libsigc.patch
+)
+
+src_configure() {
+	append-cxxflags -std=c++11
+
+	econf \
+		--disable-stlport \
+		$(use_enable static-libs static)
```

Yes, this is very short, but as short as it is, it is a great place to perform a mistake in future edit :smile:

```suggestion
	local myconf=(
		--disable-stlport
		$(use_enable static-libs static)
	)
	econf "${myconf[@]}"
```

---

can we drop `static-libs` while at it? I can't find any revdeps in ::gentoo.

---

dropped `static-libs`

@arthurzam as it was down to a single argument to econf, I didn't bother with the array, but I appreciate the suggestion.